.github/workflows/weekly.yml

```
@@ -38,11 +38,20 @@ jobs:
           jdk: '8, 11, 17, 21',
           branch: main
         }, {
-          flink: 1.17.1,
-          branch: v3.0
+          flink: 1.20-SNAPSHOT,
+          jdk: '8, 11, 17, 21',
+          branch: main
+        }, {
+          flink: 1.17.2,
+          branch: v3.1
         }, {
-          flink: 1.18.0,
-          branch: v3.0
+          flink: 1.18.1,
+          jdk: '8, 11, 17',
+          branch: v3.1
+        }, {
+          flink: 1.19.0,
+          branch: v3.1,
+          jdk: '8, 11, 17',
```

```suggestion
          jdk: '8, 11, 17, 21',
```
shouldn't it be 21 as well?

---

Technically yes, but weeklys actually can't be run from non `main` branches, so will need to be fixed there.